libs/sdk-embedding/src/iframe/kd.ts

```diff
@@ -1272,4 +1282,45 @@ export namespace EmbeddedKpiDashboard {
         GdcKdEventType.SetFilterParentsFailed,
         ISetFilterParentsFailedDataBody
     >;
+
+    /**
+     * Open delete dashboard dialog, user will be able to delete the current existed dashboard
```

Comment:
Typo: current existed. Should probably be "currently existing" (this is repeated also on line 1291).